packages/fxa-content-server/tests/functional/fx_browser_relier.js

```diff
@@ -191,8 +187,127 @@ registerSuite('Firefox Desktop non-sync', {
             selectors.WOULD_YOU_LIKE_SYNC.HEADER
           )
         )
-        .then(click(selectors.WOULD_YOU_LIKE_SYNC.SUBMIT))
+        .then(
+          click(
+            selectors.WOULD_YOU_LIKE_SYNC.SUBMIT,
+            selectors.CONNECT_ANOTHER_DEVICE.HEADER
+          )
+        )
         .then(testIsBrowserNotified('fxaccounts:login'));
     },
   },
 });
+
+registerSuite('Firefox Desktop non-sync - CWTS on signup', {
+  beforeEach: function() {
+    email = TestHelpers.createEmail('signupPasswordCWTS.treatment{id}');
+    return this.remote.then(clearBrowserState());
+  },
+
+  tests: {
+    'signup with no service - do not sync': function() {
+      return (
+        this.remote
+          .then(
+            openPage(EMAIL_FIRST_URL, selectors.ENTER_EMAIL.SUB_HEADER, {
+              query: {
+                forceExperiment: 'signupCode',
+                forceExperimentGroup: 'treatment',
+                forceUA: uaStrings['desktop_firefox_58'],
```

Comment:
Should we be using 71 here?

Reply:
👍 I'll update that once [these test changes land](https://github.com/mozilla/fxa/pull/3500)

Reply:
Test changes landed, and updated. Great catch!